src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Bindings/WatchableCloudBlobStream.Logger.cs

```diff
@@ -12,13 +12,13 @@ internal static class WatchableCloudBlobStreamLoggerExtension
           LoggerMessage.Define<string, string, string, TimeSpan, long>(
               LogLevel.Debug,
               new EventId(2, nameof(BlobWriteAccess)),
-              "BlobWriteAccess - Name: {name}, Type: {type}, ETag: {etag}, WriteTime: {writeTime}, BytesWritten: {bytesWritten}");
+              "BlobWriteAccess - Name: {blobName}, Type: {type}, ETag: {etag}, WriteTime: {writeTime}, BytesWritten: {bytesWritten}");
```

Comment:
@brettsam  - do you recommend explicitly passing in the functioName to these logs? Will function execution logger scope take care of passing the function Name?

Reply:
If these logs are being emitted from input or output bindings, I believe the scope will capture the function name. For triggers it will not. You may need to do a quick test to check.